src/Time/Slab.hpp

```diff
@@ -0,0 +1,125 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class Slab
+
+#pragma once
+
+#include <functional>
+#include <limits>
+
+#include "ErrorHandling/Assert.hpp"
+
+class Time;
+class TimeDelta;
+
+namespace PUP {
+class er;
```

Comment:
The thing which pup's is 'er' in the namespace PUP, so when you use it it's PUP::er...